source/includes/_tipser-tools.md

```diff
@@ -1,3 +1,29 @@
 # Tipser Tools
 
-//tools desc
\ No newline at end of file
+Tipser enables commerce on almost any site. An easy way to enable shopping on your site is to install the Tipser script and use the Tipser Tools to insert `Get The Look` widget directly into your content. The installation of the script on your site typically takes less than 5 minutes, and can be done by even non-technical people. 
```

Comment:
I don't hear the term `Get The Look` too often recently. Maybe it's good to check if we prefer to say "insert Get The Look" or just "insert products" to the article.

Reply:
I added the question to your document. IMHO it would be less confusing to use GTL instead of insert-whatever. Because the effect is the same and the user doesn't have to know the "engine" is different.

Reply:
I see your point! We'll discuss it for sure.

Reply:
But is it okay if we don't use that term (Get THe look) nowhere in the UI? Especially not in Tipser Tools ?

Reply:
@chmurson  You are right. I think it's not okay to use the term GTL. Instead, we could use the phrase: `to insert Tipser products and buy buttons directly into your content` (this is the phrase I often hear Swedes are using when doing their pitch).

Reply:
Sorry I just realized I gave you two different suggestions for the same piece of text ;-). Feel free to pick the one that you like more ;-).

Reply:
I like your last suggestion! :D